config/puma.rb

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env puma
+
+stdout_redirect '/cellect_panoptes/log/puma_stdout.log', '/cellect_panoptes/log/puma_stderr.log'
```

Comment:
do we want this to append? if so we need the true flag at the end https://github.com/puma/puma/blob/4967226031cf5329cd7cddfe10b8a198026023f2/examples/config.rb#L54